rllib/algorithms/impala/impala.py

```diff
@@ -795,7 +795,8 @@ def place_processed_samples_on_learner_queue(self) -> None:
 
     def process_trained_results(self) -> ResultDict:
         # Get learner outputs/stats from output queue.
```

Comment:
The comment is outdated here. Could you explain via some more one-line comments what we do here?